### PR TITLE
[ADD] product_shelf_info

### PR DESCRIPTION
**Pull request by TimLai125**
#[845](https://www.quartile.co/web?debug=#id=845&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by yostashiro** [comment]
@TimLai125 I guess you are still updating this PR, right?  Please set it to draft and let me know once it's ready for a review.

**Comment by TimLai125** [comment]
@yostashiro Sorry that I just realise I should make some further adjustments, I will put the PR to draft for now.

**Comment by TimLai125** [comment]
@yostashiro The PR is ready to be reviewed.

File: product_shelf_management/report/report_deliveryslip.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="report_delivery_document" inherit_id="stock.report_delivery_document">
+        <xpath expr="//span[@t-field='move.product_id']" position="before">
+            <t>保管: <span t-field="move.product_id.shelf_id" /></t>
```
Comment (yostashiro) [inline product_shelf_management/report/report_deliveryslip.xml]: Why don't we stick to the practice of having English as the source string and add the translation?

**Review by yostashiro — CHANGES_REQUESTED** [review]
This is a non-blocking issue, but it may be better to make the module name "product_shelf_info" or something like that.  "*_management" sounds like a major function which this module is not.

File: product_shelf_management/report/report_deliveryslip.xml (hunk 1 above)
Comment (TimLai125) [reply product_shelf_management/report/report_deliveryslip.xml]: OK, I have adjusted to the standard practice.

**Comment by TimLai125** [comment]
@yostashiro I have re-named the module as well

**Review by yostashiro — APPROVED** [review]
LGTM!

**Pull request merged**